config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,16 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+#: --tmpfs parameter
+tmpfs_path = /run
+run_options_csv = --tmpfs %(tmpfs_path)s
+#: command to run in container to check if passed --tmpfs is mounted as tmpfs type
+fndmnt = findmnt -n -t tmpfs %(tmpfs_path)s
+cmd = 'sleep 2s && %(fndmnt)s'
+run_append_name = True
```

Comment:
>Should I also remove exit_status?
￼
That's a more interesting question, because exit_status is vital to understanding the test. So even though it's duplication to list it explicitly, it _might_ make it slightly quicker for a future reader to understand. My (slight) preference is to remove it, but the existing convention in `run.ini` is clearly to reiterate it in each subsubtest -- so I would recommend leaving it, in deference to convention.

But I don't have a strong preference either way, and will accept whatever you choose.